cmd/dfdaemon/app/root_test.go

```diff
@@ -51,6 +51,43 @@ func (ts *rootTestSuite) TestConfigNotFound() {
 	r.True(os.IsNotExist(errors.Cause(readConfigFile(v, rootCmd))))
 }
 
+func (ts *rootTestSuite) TestNodeFlag() {
+	r := ts.Require()
+	fs := afero.NewMemMapFs()
+
+	fs.Create("/dfget")
+
+	configName := "dfdaemon.yml"
+	file, err := fs.Create(configName)
+	r.Nil(err)
+	file.WriteString("supernodes:\n- 127.0.0.1:6666")
+	file.Close()
+	rootCmd.Flags().Set("config", configName)
```

Comment:
And could you please add more test cases:

1. use config file that is not exist and specify the node with cli flag
2. neither use config file and cli flag to specify the node and the expected result should be `127.0.0.1:8002`

Reply:
Currently there's no default value for `supernodes`. Should we add one?